examples/plot_unified_validation_scheme.py

```diff
@@ -0,0 +1,344 @@
+
+This tutorial shows you how to properly train, tune and test your deep learning
+models with Braindecode. We will use the BCIC IV 2a dataset as a showcase example,
+however this scheme holds for any supervised trial-based decoding setting.
+As this tutorial will include additional parts of code like loading and preprocessing,
+defining a model, etc. which are not exclusive to this page (compare `Cropped Decoding
+Tutorial <./plot_bcic_iv_2a_moabb.html>`__), feel free to skip these parts.
+In general we distinguish between 3 different schemes, one for the final training and two
+different methods for tuning/hyperparameter search.
+
+"""
+
+######################################################################
+# Why should I care about that?
+# -------------------------------------
+# Short answer: To produce reliable results.
+# To train a Machine Learning model you typically use two distinct
+# datasets: one for training and one for testing. Easy - right?
+# But the story does not end here. While developing a ML model you
+# always have to adjust and tune the Hyperparamters of your model/
+# pipeline (i.e. number of layers, learning rate, number of epochs).
+# If you would keep using the test dataset to evaluate your adjustment
+# you would run into something called data leakage. This means that,
+# by using the test set to adjust the Hyperparamters of your model,
+# the model implicitly learns from the test set. Therefore the trained
+# model is not independent of the test set anymore (even though they
+# were never used for backpropagation!).
+# This is why you need a third split, the so called validation set, if
+# you perform any Hyperparamter tuning.
+# This tutorial shows two different methods (Option 2 and 3) to do
+# Hyperparamter tuning.
+# Option 1 shows how to train with a 2-fold split (train and test,
+# no validation split). Option 1 should only be used if you already
+# know your Hyperparamter configuration.
+#
+
+######################################################################
+# Loading and preprocessing the dataset
+# -------------------------------------
+#
+
+
+######################################################################
+# Loading
+# ~~~~~~~
+#
+
+from braindecode.datasets import MOABBDataset
+
+subject_id = 3
+dataset = MOABBDataset(dataset_name="BNCI2014001", subject_ids=[subject_id])
+
+
+######################################################################
+# Preprocessing
+# ~~~~~~~~~~~~~
+#
+
+from braindecode.preprocessing import (
+    exponential_moving_standardize, preprocess, Preprocessor, scale)
+
+low_cut_hz = 4.  # low cut frequency for filtering
+high_cut_hz = 38.  # high cut frequency for filtering
+# Parameters for exponential moving standardization
+factor_new = 1e-3
+init_block_size = 1000
+
+preprocessors = [
+    Preprocessor('pick_types', eeg=True, meg=False, stim=False),  # Keep EEG sensors
+    Preprocessor(scale, factor=1e6, apply_on_array=True),  # Convert from V to uV
```

Comment:
I would use `np.multiply` here to avoid `DeprecationWarning` in the doc

Reply:
Agree

Reply:
Do you know how to properly do that? The problem with `np.multiply` is that x2 (the factor) is a positional argument and not a keyword argument. Therefore
`Preprocessor(np.multiply, x2=1e6, apply_on_array=True)`
does not work. My current workaround would look like this:
`Preprocessor(lambda data, factor: np.multiply(data, factor), factor=1e6)`

Reply:
good question! :smile:  I just followed the DeprecationWarning message. I think we should remove the DeprecationWarning and abandon the plan of removing `scale` function because in our flow `np.multiply` can't replace it. At least I don't see an easy way as we use keyword arguments in preprocessing (the lambda you've proposed does exactly the same as current `scale` function). 

We may remove the DeprecationWarning here or in another PR if everyone agrees that it won't work with `np.multiply` @robintibor @hubertjb @bruAristimunha

Reply:
Sure @sliwy. After finishing this tutorial we will do a PR to update the warning message.